diagnostic/conflict.go

```diff
@@ -72,6 +77,36 @@ func groupConflicts(allConflicts []conflict) []conflict {
 			key = p.producerRepr + ";" + p.consumerRepr
 			if p.producerPosition.IsValid() {
 				key = p.producerPosition.String() + ": " + p.producerRepr
+			} else {
```

Comment:
Can we add some comments here listing an example where we would hit this case for future readers? :)

Reply:
Done :)